### PR TITLE
fix(sharing): remove message for shared links for Canvas files

### PR DESCRIPTION
**Pull request by azotov-box**
Remove "login required" message for shared links for Canvas files since we are releasing anonymous access to Canvas

### CODE REVIEW

**Review by bartoszswitalski — APPROVED** [review]

**Comment by greg-in-a-box** [comment]
@azotov-box did you use the `git revert` command to perform the revert?

**Comment by azotov-box** [comment]
> @azotov-box did you use the `git revert` command to perform the revert? the description doesnt look like any of the other reverted commits via the command

No, I did not. I updated the description

File: src/features/unified-share-modal/SharedLinkSection.js [hunk 1]
```diff
@@ -331,7 +331,6 @@ class SharedLinkSection extends React.Component<Props, State> {
                         allowedAccessLevels={allowedAccessLevels}
                         changeAccessLevel={changeSharedLinkAccessLevel}
                         enterpriseName={enterpriseName}
-                        item={item}
```
Comment (tjuanitas) [inline src/features/unified-share-modal/SharedLinkSection.js:334]: there's a `TODO` in this file around line ~388 to ~406 that uses `isBoxCanvasFile`. Do we still need those changes?

**Review by tjuanitas — COMMENTED** [review]
Can we remove this message ID? `canvasSharedLinkPubliclyAvailable`

File: src/features/unified-share-modal/SharedLinkSection.js (hunk 1 above)
Comment (azotov-box) [reply src/features/unified-share-modal/SharedLinkSection.js:334]: Done

**Review by tjuanitas — APPROVED** [review]

**Review by kolosovsky — APPROVED** [review]

**Pull request merged**